### ISSUE TITLE
Module 1, Page 5: Concrete Model Activity

### ISSUE THREAD

**Issue by mcs160** [bke858/math-whiz#45]
"On this page, you will begin by reviewing each of the four problems shown on the screen and draw a picture equivalent using a pizza pie either on a separate sheet of paper or whiteboard. Once the problem has been solved, hover over the result to see if your answer matches."

Parallelism error between "reviewing" and "draw".  suggest changing "draw" to "drawing" or rewording.

This is a very long and complex sentence, I suggest rewording.

"On this page, you will begin by reviewing each of the four problems shown on the screen.  For each problem, draw a picture or pizza pie showing the problem with an answer.

Once completed, hover over the problem to see if yours matches."

**Comment by ylp290** [comment]
fixed.
